library/common/1.0.0/docs/container/securityContext.md

```diff
@@ -57,3 +57,10 @@ workload:
               drop:
                 - ALL
 ```
+
+---
+
+Notes:
+
+When setting capabilities for containers, remember to **NOT** include `CAP_` prefix.
+For example, `CAP_NET_ADMIN` should be `NET_ADMIN`.
```

Comment:
No reservations but might be more confusing for contributors?

Reply:
That's how kubernetes wants them defined. Took me a while to remember it and documented it.